packages/densityscout/PKGBUILD

```diff
@@ -0,0 +1,30 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=densityscout
+pkgver=45
+pkgrel=1
+pkgdesc='This tool calculates density for files of any file-system-path to finally output an accordingly descending ordered list.'
+arch=('x86_64' 'aarch64')
+groups=('blackarch' 'blackarch-misc')
+url='https://www.cert.at/en/downloads/software/software-densityscout'
+license=('custom:cert.at')
+source=("https://www.cert.at/media/files/downloads/software/densityscout/files/${pkgname}_build_45_linux.zip")
+sha512sums=('SKIP')
+
+package() {
+  install -Dm 644 "license.txt" "$pkgdir/usr/share/licenses/$pkgname/LICENSE"
+
+  if [ $(uname -m) == "x86_64" ]; then
+    cd lin64
+
+    install -Dm 644 "$pkgname" "$pkgdir/usr/bin/$pkgname"
```

Comment:
you can use only one `install -Dm 644 "$pkgname" "$pkgdir/usr/bin/$pkgname"` outside of the if/else block since it's the same in both cases